# Generated by Django 3.2.13 on 2023-09-08 12:45

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('db', '0065_auto_20230906_1304'),
    ]

    operations = [
        migrations.CreateModel(
            name='ReasonForError',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='Created At')),
                ('updated_at', models.DateTimeField(auto_now=True, verbose_name='Last Modified At')),
                ('name', models.CharField(max_length=255)),
                ('description', models.TextField(blank=True)),
                ('topic', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='reason_for_error', to='db.topic')),
            ],
            options={
                'verbose_name': 'ReasonForError',
                'verbose_name_plural': 'ReasonForErrors',
                'db_table': 'reason_for_errors',
            },
        ),
    ]
